### ISSUE TITLE
Changes / Updates / Dates for Round 19

### ISSUE THREAD

**Issue by nbrady-techempower** [TechEmpower/FrameworkBenchmarks#4966]
I'll be editing this pinned issue to link to other issues or pull requests that may affect tests for this round.

No set dates yet for Round 19.

**Comment by bhauer** [comment]
* #5015 The requirements are not just on GitHub, and the results web site links there instead of having a duplicate copy.

**Comment by zloster** [comment]
Hi all. I want to bring attention to the following issues:

1. There is no response for quite long time on these two: https://github.com/TechEmpower/FrameworkBenchmarks/issues/4092#issuecomment-484549733 and https://github.com/TechEmpower/FrameworkBenchmarks/issues/4885#issuecomment-506957575 I'm not able to pinpoint the problem that is causing the described behaviour.
1. https://github.com/TechEmpower/FrameworkBenchmarks/issues/4083#issuecomment-549923026 I would like to hear how to proceed with this. It concerns the frameworks using `Java` and `servlet` container. The data suggests there is a difference.

**Comment by volyrique** [comment]
Are there any plans for OS upgrades? Newer kernel versions might be affected differently by the security countermeasures that led to a lot of performance regressions before. CentOS 8 got released more than a month ago (CentOS 7 is used in the physical hardware environment), and I believe that we have more recent options for the Azure VMs (which use Ubuntu 16.04 if I am not mistaken).

**Comment by bhauer** [comment]
@volyrique Indeed, we are presently considering switching over to Ubuntu 18.04.

**Comment by hushino** [comment]
Update package.json of Hapi from ` "hapi": "16.1.1"` to `"@hapi/hapi": "^18.4.0" `
[package.json of hapi tests](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/JavaScript/hapi/package.json)

**Comment by volyrique** [comment]
How about any plans to visualize the cached queries test type?

**Comment by bhauer** [comment]
@volyrique Good question. I'll see if I can get that enabled for the next round.

**Comment by cricket007** [comment]
Please upgrade Dropwizard to at least 1.3.17 or to 2.0

https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Java/dropwizard/pom.xml#L17

**Comment by cricket007** [comment]
Please upgrade Quarkus to a 1.x.y release 

https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Java/quarkus/pom.xml#L10

**Comment by Liroo** [comment]
Adding uWebSocket @alexhultman

**Comment by nbrady-techempower** [comment]
Round 19 will run in April. Pull requests not ready by 4/3 won't be guaranteed to be part of Round 19.

**Comment by zloster** [comment]
@nbrady-techempower @bhauer 
https://github.com/TechEmpower/FrameworkBenchmarks/issues/4092#issuecomment-551295713
The problem with the TCP/IP connections is still there and there is no comment from your side. I've found a possible reproduction steps. It's just a configuration change.
Please, take a closer look.

**Comment by nbrady-techempower** [comment]
We're taking Citrine down for maintenance and working on some pressing issues that may have a negative impact on the next round. We'll be postponing round 19 for a couple of weeks, so please feel free to keep submitting pull requests. We'll have messaging on Citrine and in this thread when things are back up.

**Comment by sebastienros** [comment]
> pressing issues that may have a negative impact on the next round

Can you share the details if we want to repro the environment?

**Comment by msmith-techempower** [comment]
> Can you share the details if we want to repro the environment?

We will share any changes we make.

**Comment by nbrady-techempower** [comment]
@sebastienros We're working more closely on this issue right now https://github.com/TechEmpower/FrameworkBenchmarks/issues/4092 and anything else we do will be completely transparent.

**Comment by cricket007** [comment]
I would contribute a PR, but tied up with other priorities. Can someone take care of Dropwizard, please?

**Comment by joanhey** [comment]
Before close the PRs for the next Round.

Please add a message in advance in https://tfb-status.techempower.com/
Like the one that you place when you're doing maintenance.
![DeepinScreenshot_Seleccionar área_20200410003104](https://user-images.githubusercontent.com/249085/78946386-dab8a700-7ac2-11ea-97b5-31dcc599f9a4.png)

**Comment by nbrady-techempower** [comment]
Thank you @joanhey for making that request here. I've added it to the top as a reminded to myself.

**Comment by nbrady-techempower** [comment]
We're moving forward with Round 19:

Round 19 testing will begin on or around May 1st. Last call for changes to be reviewed and merged is April 29th.

**Comment by joanhey** [comment]
Update Azure to Ubuntu 18.04 like in Citrine.

**Comment by nbrady-techempower** [comment]
@joanhey Please don't duplicate all your comments. You're driving my notifications crazy! :(

**Comment by Phara0h** [comment]
Are you hard set on the deadline for today? I was planning on having mine by the end of the weekend. Pretty sure I have the fastest web framework on the web right now, minium fastest with node.js.

**Comment by nbrady-techempower** [comment]
@Phara0h A continuous run started today on Citrine. The next run will be reviewed for Round 20, so technically you do have a few days, but it would not give you any time to review and fix data.

**Comment by nbrady-techempower** [comment]
Round 19 results have been posted: Round 19 results have been posted: https://www.techempower.com/benchmarks/#section=data-r19

Unpinning this issue but will leave open for a bit. Pinning a new issue to start tracking changes for Round 20

**Comment by NateBrady23** [comment]
Closing as part of bulk cleanup.
